### ISSUE TITLE
Timetable validation viewer

### ISSUE THREAD

**Issue by landonreed** [conveyal/datatools-ui#307]
<a href="https://github.com/evansiroky"><img src="https://avatars1.githubusercontent.com/u/3112493?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [evansiroky](https://github.com/evansiroky)**
_Saturday Jun 16, 2018 at 01:18 GMT_
_Originally opened as https://github.com/catalogueglobal/datatools-ui/pull/198_

----

Show a preview of timetables in the feed viewer

----

_**[evansiroky](https://github.com/evansiroky)** included the following code: https://github.com/catalogueglobal/datatools-ui/pull/198/commits_

**Comment by landonreed** [comment]
<a href="https://github.com/codecov-io"><img src="https://avatars0.githubusercontent.com/u/8655789?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [codecov-io](https://github.com/codecov-io)**
_Saturday Jun 16, 2018 at 01:18 GMT_

----

# [Codecov](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198?src=pr&el=h1) Report
> Merging [#198](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198?src=pr&el=desc) into [dev](https://codecov.io/gh/catalogueglobal/datatools-ui/commit/b8be4e24a69e8db298885c50b0220ef37f69a77c?src=pr&el=desc) will **increase** coverage by `0.09%`.
> The diff coverage is `5.64%`.

[![Impacted file tree graph](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198/graphs/tree.svg?width=650&height=150&src=pr&token=P6y8RDP2dF)](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198?src=pr&el=tree)

```diff
@@           Coverage Diff            @@
##             dev    #198      +/-   ##
========================================
+ Coverage   0.03%   0.13%   +0.09%     
========================================
  Files        316     321       +5     
  Lines      10932   11144     +212     
  Branches    3846    3913      +67     
========================================
+ Hits           4      15      +11     
- Misses      8953    9129     +176     
- Partials    1975    2000      +25
```


| [Impacted Files](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [lib/gtfs/util/graphql.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198/diff?src=pr&el=tree#diff-bGliL2d0ZnMvdXRpbC9ncmFwaHFsLmpz) | `0% <ø> (ø)` | :arrow_up: |
| [lib/editor/util/timetable.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198/diff?src=pr&el=tree#diff-bGliL2VkaXRvci91dGlsL3RpbWV0YWJsZS5qcw==) | `0% <ø> (ø)` | :arrow_up: |
| [lib/gtfs/actions/feed.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198/diff?src=pr&el=tree#diff-bGliL2d0ZnMvYWN0aW9ucy9mZWVkLmpz) | `0% <0%> (ø)` | :arrow_up: |
| [lib/gtfs/actions/filter.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198/diff?src=pr&el=tree#diff-bGliL2d0ZnMvYWN0aW9ucy9maWx0ZXIuanM=) | `0% <0%> (ø)` | :arrow_up: |
| [lib/gtfs/reducers/timetables.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198/diff?src=pr&el=tree#diff-bGliL2d0ZnMvcmVkdWNlcnMvdGltZXRhYmxlcy5qcw==) | `0% <0%> (ø)` | |
| [lib/manager/components/version/FeedVersionTabs.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198/diff?src=pr&el=tree#diff-bGliL21hbmFnZXIvY29tcG9uZW50cy92ZXJzaW9uL0ZlZWRWZXJzaW9uVGFicy5qcw==) | `0% <0%> (ø)` | :arrow_up: |
| [lib/manager/selectors/index.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198/diff?src=pr&el=tree#diff-bGliL21hbmFnZXIvc2VsZWN0b3JzL2luZGV4Lmpz) | `0% <0%> (ø)` | :arrow_up: |
| [lib/gtfs/reducers/index.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198/diff?src=pr&el=tree#diff-bGliL2d0ZnMvcmVkdWNlcnMvaW5kZXguanM=) | `0% <0%> (ø)` | :arrow_up: |
| [.../components/reporter/components/TimetableLayout.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198/diff?src=pr&el=tree#diff-bGliL21hbmFnZXIvY29tcG9uZW50cy9yZXBvcnRlci9jb21wb25lbnRzL1RpbWV0YWJsZUxheW91dC5qcw==) | `0% <0%> (ø)` | |
| [lib/gtfs/reducers/filter.js](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198/diff?src=pr&el=tree#diff-bGliL2d0ZnMvcmVkdWNlcnMvZmlsdGVyLmpz) | `0% <0%> (ø)` | :arrow_up: |
| ... and [13 more](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198/diff?src=pr&el=tree-more) | |

------

[Continue to review full report at Codecov](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198?src=pr&el=footer). Last update [b8be4e2...16e1ca8](https://codecov.io/gh/catalogueglobal/datatools-ui/pull/198?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by landonreed** [comment]
<a href="https://github.com/landonreed"><img src="https://avatars2.githubusercontent.com/u/2370911?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [landonreed](https://github.com/landonreed)**
_Monday Jul 02, 2018 at 20:18 GMT_

----

Closing because fixes for this were applied in #202.
